backend/app/services/system_skill_providers/service.py

```
@@ -438,28 +461,63 @@ def _find_skill(
+        *,
+        user_id: int,
+        skill_id: int,
+    ) -> Optional[Kind]:
+        return (
+            db.query(Kind)
+            .filter(
+                Kind.id == skill_id,
+                Kind.user_id == user_id,
+                Kind.kind == "Skill",
+                Kind.namespace == "default",
+                Kind.is_active == True,
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)